server/command/install.go

```diff
@@ -40,7 +40,36 @@ func (s *service) executeInstall(params *params) (*model.CommandResponse, error)
 	if err != nil {
 		return errorOut(params, err)
 	}
+	return s.installApp(manifest, appSecret, force, params)
+}
+
+func (s *service) executeInstallAWSApp(params *params) (*model.CommandResponse, error) {
```

Comment:
Is there anything AWS-specific invoked here?

Reply:
Correct there is nothing else aws specific. We install with `app-id` or with the manifest URL.

Reply:
Then prob. we should change the name of the method from `executeInstall**AWS**App`

Reply:
Should we also rename `executeInstall` to `executeInstallHTTPApp` then and also change the slash command name?